fieldservice_agreement_helpdesk/views/fsm_order.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- FieldService Order Form View -->
+    <record id="view_fsm_order_ticket_form" model="ir.ui.view">
+        <field name="name">view.fsm.order.ticket.form</field>
+        <field name="model">fsm.order</field>
+        <field name="inherit_id" ref="fieldservice.fsm_order_form"/>
+        <field name="arch" type="xml">
+            <field name="agreement_id" position="attributes">
+                <attribute name="context">{'default_agreement_id': agreement_id}</attribute>
```

Comment:
@osi-scampbell There is no agreement_id field on agreement. You probably want to set the customer field of the agreement:

```
<attribute name="context">{'default_partner_id': partner_id}</attribute>
```